bin/critterget.py

```diff
@@ -269,6 +269,57 @@ def getCrashSummary(appId, appName):
     return CrashDict
 
 
+def get_exception_summary(app_id, app_name):
+    # given an appID and an appName, produce a Splunk KV summary of the
+    # exceptions for a given timeframe
+
+    mystime = scopetime()
+    exception_data = None
+
+    http_code = None
+    retry = 0
+    while http_code != 200:
+        http_code, exception_data = apicall_with_response_code(
+            "app/exception/summaries/{}".format(
+                app_id
+            ),
+            {'lastOccurredStart': mystime}
+        )
+        retry += 1
+        if retry > MAX_RETRY:
+            break
+
+    all_exceptions = {}
+    if http_code != 200:
+        print (u'{} MessageType="ExceptionSummary" appId={} appName="{}" '
+               u'Could not get exception summaries for {}. '
+               u'Code: {} after retry {}').format(DATETIME_OF_RUN,
+                                                  app_id, app_name,
+                                                  mystime,
+                                                  http_code,
+                                                  retry)
+    elif exception_data:
+        for i, y in enumerate(exception_data[DATA]):
```

Comment:
I'm not sure why enumerate is being used here.  Couldn't you just iterate over exception_data[DATA] ?